core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2690,7 +2690,10 @@ object SparkContext extends Logging {
         setActiveContext(new SparkContext(config))
       } else {
         if (config.getAll.nonEmpty) {
-          logWarning("Using an existing SparkContext; some configuration may not take effect.")
+          logWarning("Using an existing SparkContext; some configuration may not take effect." +
+            " For how to set these configuration correctly, you can refer to" +
```

Comment:
nit: for following the other formats in this file, could you move the leading single space into the previous line?
```
          logWarning("Using an existing SparkContext; some configuration may not take effect. " +
            "For how to set these configuration correctly, you can refer to" +
```

Reply:
> nit: for following the other formats in this file, could you move the leading single space into the previous line?
> 
> ```
>           logWarning("Using an existing SparkContext; some configuration may not take effect. " +
>             "For how to set these configuration correctly, you can refer to" +
> ```

Done